### PR TITLE
fix(keyword description): add description and keyword editing workflow

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

File: src/components/icons/GoldPlus.svg
Comment (atticusofsparta) [inline src/components/icons/GoldPlus.svg:1]: I normally wouldnt pull in an icon like this, but this is a special case due to the wierd gradient it has set up

File: src/components/forms/DomainSettings/KeywordRow.tsx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+          />{' '}
+          <span className="flex flex-row max-w-fit h-fit font-semibold items-center">
+            add
+          </span>
+        </button>
+      ) : (
+        <div className="flex flex-row max-w-[330px]" style={{ gap: '10px' }}>
+          <input
+            ref={inputRef}
+            className={`flex flex-row rounded-full bg-background border border-dark-grey shadow-3xl py-[2px] items-center justify-center px-[10px] w-fit w-fit text-white transition-all`}
```
Comment (kunstmusik) [inline src/components/forms/DomainSettings/KeywordRow.tsx:66]: Change px to rem

Comment (kunstmusik) [reply src/components/forms/DomainSettings/KeywordRow.tsx:66]: apply throughout PR

**Review by kunstmusik — COMMENTED** [review]
Should change all of the px values to rem. (I know, a bit of a pain, but it’ll make it look good if someone zooms the page or has a different default font setting)

File: src/components/forms/DomainSettings/KeywordRow.tsx (hunk 1 above)
Comment (atticusofsparta) [reply src/components/forms/DomainSettings/KeywordRow.tsx]: [2b04cdf](https://github.com/ar-io/arns-react/pull/620/commits/2b04cdff991e2f1b828fef7102a1b73d938e65ee)

**Review by kunstmusik — DISMISSED** [review]

**Pull request merged**